src/lib/Elf.cc

```diff
@@ -73,37 +77,36 @@ Elf::Elf(std::string path, char** imagePointer) {
    */
 
   // Seek to the byte representing the start of the header offset table.
-  uint64_t headerOffset;
-  file.read(reinterpret_cast<char*>(&headerOffset), sizeof(headerOffset));
+  uint64_t e_phoff;
```

Comment:
Although this is now inline with the linux implementation, it is much less readable. I think the name should be reverted back to `headerOffset`

Reply:
I disagree, when working alongside the linux documentation and man pages this makes it much easier to follow what variables in SimEng correspond to. Once used to the naming convention it is also more descriptive of what that variable holds e.g. e_phoff holds the "P"rogram "H"eader "OFF"set that was held in the "E"lf header. Comments in the header should now mitigate initial decrease in readability